downloadCurrentVersion.js

```diff
@@ -3,6 +3,8 @@
 // This is used to download the correct binary version
 // as part of the prepublish step.
 
+const SOLC_REPO = process.env.SOLC_REPO || 'https://raw.githubusercontent.com/ethereum/solc-bin/gh-pages/bin/';
```

Comment:
What is the use case for changing the official URL here?

Reply:
The GitHub pages URL becomes rate limited once you exceed certain bandwidth allowance, which made our CI builds fail when downloading solc. Using the raw URL bypasses the pages rate limits.

Reply:
I understand that, but not the reason for `env.SOLC_REPO`.

Reply:
Commented underneath.